src/integrationTest/java/com/mongodb/hibernate/query/select/SortingSelectQueryIntegrationTests.java

```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright 2025-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.hibernate.query.select;
+
+import static com.mongodb.hibernate.internal.MongoConstants.MONGO_DBMS_NAME;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.hibernate.cfg.QuerySettings.DEFAULT_NULL_ORDERING;
+
+import com.mongodb.hibernate.internal.FeatureNotSupportedException;
+import java.util.Arrays;
+import java.util.List;
+import org.hibernate.query.NullPrecedence;
+import org.hibernate.query.SortDirection;
+import org.hibernate.testing.orm.junit.DomainModel;
+import org.hibernate.testing.orm.junit.ServiceRegistry;
+import org.hibernate.testing.orm.junit.Setting;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+@DomainModel(annotatedClasses = Book.class)
+class SortingSelectQueryIntegrationTests extends AbstractSelectionQueryIntegrationTests {
+
+    private static final List<Book> testingBooks = List.of(
+            new Book(1, "War and Peace", 1869, true),
+            new Book(2, "Crime and Punishment", 1866, false),
+            new Book(3, "Anna Karenina", 1877, false),
+            new Book(4, "The Brothers Karamazov", 1880, false),
+            new Book(5, "War and Peace", 2025, false));
+
+    private static List<Book> getBooksByIds(int... ids) {
+        return Arrays.stream(ids)
+                .mapToObj(id -> testingBooks.stream()
+                        .filter(c -> c.id == id)
+                        .findFirst()
+                        .orElseThrow(() -> new IllegalArgumentException("id does not exist: " + id)))
+                .toList();
+    }
+
+    @BeforeEach
+    void beforeEach() {
+        getSessionFactoryScope().inTransaction(session -> testingBooks.forEach(session::persist));
+        getTestCommandListener().clear();
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"ASC", "DESC"})
+    void testOrderBySingleFieldWithoutTies(String sortDirection) {
+        assertSelectionQuery(
+                "from Book as b ORDER BY b.publishYear " + sortDirection,
+                Book.class,
+                "{ 'aggregate': 'books', 'pipeline': [ { '$sort': { 'publishYear': "
```

Comment:
Let's use a nicely-formatted multi-line string for all of these.

Since you are using parameters, let's use `.formatted(` to set the parameter in the string (you do this in other tests).

Reply:
good point! Committed at https://github.com/mongodb/mongo-hibernate/pull/90/commits/804216351d1df7432e399357a1d04f51f4d0e040

Will create a quick PR without ticket later to fix similar issue in existing `SimpleSelectQueryIntegrationTests`.

Reply:
Given the refactoring in `SimpleSelectQueryIntegrationTests` is easy to verify, I included it at https://github.com/mongodb/mongo-hibernate/pull/90/commits/a2cef52ffdc86beb04099cdc2f48356ceab95a55

Reply:
Note that another `palantir` formatting issue was flushed out (`.formatted()` will be formatted in a new line, not right after the ending triple quotes of the text block). I solved this format issue at another PR: https://github.com/mongodb/mongo-hibernate/pull/97

Reply:
I responded in 97. I don't think that we need to change the position of the `.formatted` call, but this is outside of the scope of this PR anyway.